[#543] Incorrect exception message in ViewLoaderReflectionUtils

nils-christian opened the pull request:
This is a pull request for #543. I corrected the exception message as originally discussed. I also made sure that the ScopeProvider has _value_ as an alias for _scopes_, which is handled in the _ViewLoaderReflectionUtils_. From the two view models in the example using a scope provider, one uses now _scopes_ and the other one the _value_ field.

Commit:
[#543] Incorrect exception message in ViewLoaderReflectionUtils

Signed-off-by: Nils Christian Ehmke <nils@rhocas.de>

manuel-mauky commented:
Many thanks for your contribution. There is just a small issue I have: In the `getScopesFromProvider` method you are using some code formatting settings that don't match the rest of the code. 
You are using tabs instead of (4) spaces for indentation and you are placing spaces in method parentheses (i.e. `someMethod( 42 )` instead of `someMethod(42)`). 

It would be nice if you could fix that before I merge the PR. 

We also have an [eclipse formatter config](https://github.com/sialcasa/mvvmFX/blob/develop/mvvmfx_formatter.xml) but I have to admit that I'm not sure if it is used consistently across the whole code base and as I'm personally using IntelliJ instead of eclipse it's likely that I'm commiting code that doesn't match this config from time to time. I'm not fundamental with coding guidelines but I try to keep the basics like indentation as clean as possible.

Commit:
[#543] Incorrect exception message in ViewLoaderReflectionUtils

Signed-off-by: Nils Christian Ehmke <nils@rhocas.de>

nils-christian commented:
Hi,

Sure. I corrected the indentation and removed the spaces before and after the parentheses. I also corrected a minor typo in the Javadoc.

Best regards

  Nils
